packages/rmf-codegen/scripts/install-vercel-java.js

```diff
@@ -0,0 +1,10 @@
+const shelljs = require('shelljs');
+const { javaIsInstalled, abortIfError } = require('./helpers');
+
+if (!javaIsInstalled() && shelljs.which('amazon-linux-extras')) {
+  console.info('Installing Java on Vercel.');
```

Comment:
Where do you check that this script only runs in Vercel? 🧐

Reply:
I don't know how to check that, but the log is an indicator that this is expected to run on Vercel.

Reply:
Then mentioning Vercel in the log does not make any sense, sorry.

Reply:
I disagree.

Do you want the log removed?

Reply:
No, simply don't mention Vercel.

Reply:
If you want to keep this specific to Vercel, we should also additionally check for the `VERCEL` environment variable.

Then having the script running only on Vercel does make sense to have.